central/authprovider/service/service_impl.go

```diff
@@ -158,37 +158,37 @@ func (s *serviceImpl) GetAuthProviders(_ context.Context, request *v1.GetAuthPro
 func (s *serviceImpl) PostAuthProvider(ctx context.Context, request *v1.PostAuthProviderRequest) (*storage.AuthProvider, error) {
 	providerReq := request.GetProvider()
 	if providerReq.GetName() == "" {
-		return nil, errox.NewErrInvalidArgs("no auth provider name specified")
+		return nil, errox.InvalidArgs.CausedBy("no auth provider name specified")
 	}
 	if providerReq.GetId() != "" {
-		return nil, errox.NewErrInvalidArgs("auth provider id must be empty")
+		return nil, errox.InvalidArgs.CausedBy("auth provider id must be empty")
```

Comment:
Same as above. Consider adjusting the msgs for all such instances.

Reply:
@md2119 I have adjusted messages for this file. ✅ 

I understand your concerns, and they are absolutely valid! But we should restrain ourselves from further such changes because they are out of the scope of this issue, and it would just expand the PR and make the review complicated. We should instead collect such cases and make follow-ups to address them.

Reply:
> @md2119 I have adjusted messages for this file. ✅

Thanks for addressing the comments!

> I understand your concerns, and they are absolutely valid! But we should restrain ourselves from further such changes because they are out of the scope of this issue, and it would just expand the PR and make the review complicated. We should instead collect such cases and make follow-ups to address them.

Yea, you make a valid argument. Cascading changes are not encouraged. In this case, we are changing the semantics, hence, the things that the new framework applies to, or what new framework consumes, or what the framework depends become conflicting. For example, if it were a `Wrap()` func, the relevance of the string wouldn't be an issue.